.github/workflows/adocs-build.yml

```diff
@@ -0,0 +1,26 @@
+name: build adocs
+
+on:
+  push:
+    branches:
+    - master
+jobs:
+  adoc_build:
+    runs-on: ubuntu-22.04
+    name: make; mv ./build/btcphilosophy.html ./build/index.html
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: Get build container
+      id: adocbuild
+      uses: tonynv/asciidoctor-action@master
+      with:
+          program: "make; mv ./build/btcphilosophy.html ./build/index.html"
```

Comment:
Is it possible to install imagemagick (`apt install imagemagick`) prior to running `make`? That would make the images resized to more appropriate sizes. 

See https://github.com/bitcoin-dev-philosophy/btcphilosophy/blob/f87b81e93f162aac6896bdbbc5dd69dc20f747df/Makefile#L52-L57